layers/+lang/rust/funcs.el

```diff
@@ -47,14 +47,25 @@
         (lsp)
         (spacemacs/declare-prefix-for-mode 'rust-mode "ms" "switch")
         (spacemacs/set-leader-keys-for-major-mode 'rust-mode
-          "bR" 'lsp-rust-analyzer-reload-workspace
+          "bR" 'spacemacs//lsp-rust-analyzer-reload-workspace
           "ss" 'lsp-rust-switch-server))
     (spacemacs//lsp-layer-not-installed-message)))
 
 (defun spacemacs//rust-setup-lsp-dap ()
   "Setup DAP integration."
   (require 'dap-gdb-lldb))
 
+(defun spacemacs//lsp-rust-analyzer-reload-workspace ()
+  (interactive)
+  (if (->> (lsp-workspaces)
```

Comment:
Looks like `->>` requires `dash`.
It's best to avoid it or declare it as dependency explicitly.

Reply:
Looks like `dash` comes from the `spacemacs-bootstrap` layer, which implies (to me at least) that it's always available. `ivy-spacemacs-help` doesn't seem to mention any other layer explicitly depending on `dash`, from what I can tell.

Reply:
That should be okay then.